.gitlint

```diff
@@ -0,0 +1,12 @@
+[general]
+ignore=T5
+
+[title-max-length]
+line-length=72
```

Comment:
Both this and the next one are the default values, but yeah, probably better to make it clear.